src/attestation/Attestation.ts

```diff
@@ -106,6 +105,38 @@ export default class Attestation implements IAttestation {
     return Object.assign(newAttestation, obj)
   }
 
+  public claimHash: IAttestation['claimHash']
+  public cTypeHash: IAttestation['cTypeHash']
+  public owner: IAttestation['owner']
+  public revoked: IAttestation['revoked']
+  public delegationId?: IAttestation['delegationId']
+
+  /**
+   * Builds a new [[Attestation]] instance.
+   *
+   * @param requestForAttestation - A request for attestation, usually sent by a claimer.
```

Comment:
The example and method doc would need to be updated.

Reply:
noted

Reply:
Maybe this one slipped through?  :)